x-pack/plugin/core/template-resources/src/main/resources/logs@mappings-logsdb.json

```diff
@@ -0,0 +1,31 @@
+{
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        },
+        "host.name": {
```

Comment:
Notice that I added added this specific logs@mapping-logsdb.json to make sure a `host.name` field is created and no error is thrown if default sorting on `host.name` and `@timestamp` is used.

Reply:
Why is this needed? We can't assume all data streams have it, logsdb must also work when it is not set.